realm/realm-library/src/main/cpp/util.hpp

```diff
@@ -696,6 +697,29 @@ class JniBooleanArray {
     jint                m_releaseMode;
 };
 
+// Wrap jobject and call the DeleteLocalRef when destruction.
```

Comment:
Wraps jobject and automatically calls DeleteLocalRef when this object is destroyed.